src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (double) Math.round((Math.PI * radius * radius) * 10) / 10;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: "
+                + this.calculateArea()
+                + " sq. units, radius: "
+                + this.radius
+                + " units, color : "
+                + super.getColor());
```

Comment:
To improve code readability, consider creating a local variable for `calculateArea()` before the `System.out.println` statement. This way, you avoid calling the method twice (once for the area calculation and once for the output) and make the code easier to read.